IndustrialUnit.Model/Helper.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+
+namespace IndustrialUnit.Model
+{
+  public static class Helper
```

Comment:
Milyen helper? Adj neki értelmes nevet, akár szét is szedheted külön classokba, pl. PathHelper, ConversionHelper, etc.